### ISSUE TITLE
Early stopping & checkpoint selection

### ISSUE THREAD

**Issue by ercanburak** [WarranWeng/ET-Net#3]
Hi, and thanks for sharing this.

Related to #2, I wonder how you performed model checkpoint selection to obtain the model file shared [here](https://drive.google.com/file/d/1V7vj3YkbhAmgzyf6rqrkeF0HSSNwyGkO/view) and to get the quantitative results in the paper. Do you select the final model weights after the training is finished (after 300 epochs according to implementation details in the paper), or do you perform early stopping according to some criteria on a validation set, and select model weights from an earlier epoch?

**Comment by WarranWeng** [comment]
Thanks for your attention.
Sorry for the late reply. I am busy with other things recently.
We apply ealy stopping and choose the model with the best result in validation set.
